### ISSUE TITLE
learning rate seems not properly be set when training with "self.args.resume == True"

### ISSUE THREAD

**Issue by DSonDH** [cure-lab/SCINet#70]
for example of  SCINet/experiments/exp_ETTh.py,

I think 
``` python3
        if self.args.resume:
            self.model, lr, epoch_start = load_model(self.model, path, model_name=self.args.data, horizon=self.args.horizon)
        else:
            epoch_start = 0
```
should be above line 227,
``` python3
model_optim = self._select_optimizer()
```
and 
```python
self.args.lr = lr 
```
should be added above line 227.

The load_model() function brings last learning rate from loaded model successfully.
But optimizer cannot set that learning rate for optimization because optimizer is called first.
It seems optimizer only takes self.args.lr and it is not the value from the loaded model.

Best Regards,
